sycl/doc/developer/ContributeToDPCPP.md

```diff
@@ -62,6 +62,8 @@ end-to-end or SYCL-CTS tests.
 - Add a helpful comment describing what the test does at the beginning and
   other comments throughout the test as necessary.
 
+- Use variables with name length more than 1 symbol in `llvm/sycl` headers and source files. 
```

Comment:
Should this rule be applied to variable names only? I thought the problem impacts all C++ identifiers. Am I wrong?

Typical macro naming convention: all words in capital letters with "_" as a separator.
Won't it be more clear to say something like: 
"- To avoid re-definition by the pre-processor, variable names should contain at
  least one lower case letter."

I'm not sure if "more than 1 symbol" is enough to guarantee that names won't conflict.

Reply:
Yes, you are right. I'll change it.